frontend/src/cards/DisparityBarChartCard.tsx

```diff
@@ -71,19 +72,14 @@ function DisparityBarChartCardWithKey(props: DisparityBarChartCardProps) {
 
   const query = new MetricQuery(metricIds, breakdowns);
 
-  function getTitleText() {
-    return `Population vs. ${
-      metricConfig.fullCardTitleName
-    } in ${props.fips.getSentenceDisplayName()}`;
-  }
+  const chartTitle = useCreateChartTitle(
+    metricConfig.populationComparisonMetric as MetricConfig,
+    locationName
+  );
 
-  const HASH_ID: ScrollableHashId = "population-vs-distribution";
+  const filename = `${metricConfig.populationComparisonMetric?.chartTitle}${locationName}`;
```

Comment:
I've run into issues with using optional chaining inside string template literals, I think if this metricConfig doesn't have a `populationComparisonMetric.chartTitle` value then the filename would end up being`undefinedFlorida` or something like that. it might be worth declaring a chartTitle variabel first with a fallback, and then using that in the templated filename string? like `const chartTitle = metricConfig.populationComparisonMetric?.chartTitle || "Some generic title"` just a consideration